src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.ts

```diff
@@ -46,18 +48,21 @@ export class FileUploadAssessmentDashboardComponent implements OnInit {
         this.busy = true;
         const exerciseId = Number(this.route.snapshot.paramMap.get('exerciseId'));
 
-        await this.getSubmissions(exerciseId)
-            // At least one submission present. Can get exercise from first submission.
-            .then(() => {
-                const exercise = this.submissions[0].participation!.exercise!;
-                FileUploadAssessmentDashboardComponent.verifyFileUploadExercise(exercise);
-                this.exercise = exercise as FileUploadExercise;
+        this.exerciseService
+            .find(exerciseId)
+            .map((exerciseResponse) => {
+                if (exerciseResponse.body!.type !== ExerciseType.FILE_UPLOAD) {
+                    throw new Error('Cannot use Text Assessment Dashboard with non-text Exercise type.');
```

Comment:
I think this should be file upload dashboard, right?

Reply:
uh good catch, I overlooked that.
```suggestion
                    throw new Error('Cannot use File-Upload Assessment Dashboard with non-file-upload Exercise type.');
```